resources/filter.py

```diff
@@ -0,0 +1,54 @@
+"""Helper Functions to be used for sorting and filtering."""
```

Comment:
This is considered as a file creating, while this is a rename. Make sure you do it in the same commit and that git mark as a rename. This is easier for merge and rebase in case someone else is updating that file.